modules/local/mmseqs_colabfoldsearch.nf

```diff
@@ -1,34 +1,40 @@
 process MMSEQS_COLABFOLDSEARCH {
-    tag "$seq_name"
+    tag "$meta.id"
     label 'process_high_memory'
 
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'docker://athbaltzis/colabfold_proteinfold:v0.9' :
         'athbaltzis/colabfold_proteinfold:v0.9' }"
 
     input:
-    tuple val(seq_name), path(fasta)
+    tuple val(meta), path(fasta)
     path ('db/params')
     path colabfold_db
     path uniref30
     val db_load_mode
```

Comment:
So, this variable is redundant, since we pass the db_load_mode through params, right? Shall we remove it from this file and modify the colabfold subworkflow file as well?

Reply:
Yep, you are right! Actually, it is the preferred way in modules to use the config together with params to pass the arguments and not declare an input value, I removed `db_load_mode`.